rpc/common/moc_common_nodeinfo.proto

```diff
@@ -58,10 +61,11 @@ message NodeInfo {
 }
 
 message Resources {
+	reserved 3;
 	reserved 4;
+	reserved 6;
 	Processor processor = 1;
 	PhysicalMemory memory = 2;
-	GPU gpu = 3;	
 	VirtualMachineCapabilities vmCapabilities = 5;
-	repeated Gpu gpuList = 6;
+	repeated HostGPU gpuList = 7;
```

Comment:
I dont think we need to realign this to 7. Since it is just the message name changing

Reply:
It's actually a field change. message name has been changed from "GPU" (uppercase) to "HostGPU". But field number 6 was "Gpu"

Sorry for the confusion. I changed the upper-case GPU to HostGPU so that we can avoid the name confusion in the future.